### ISSUE TITLE
Exception on start

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring-contrib/alignak-app#62]
When starting the recent develop branch application:
```
Traceback (most recent call last):
  File "d:\github\alignak-app\alignak_app\notifier.py", line 84, in check_data
    def check_data(self):
KeyboardInterrupt

D:\GitHub\alignak-app>python c:\Users\mohierf\AppData\Roaming\Python\alignak_app\bin\alignak-app.py
Traceback (most recent call last):
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\configparser.py", line 786, in get
    value = d[option]
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\collections\__init__.py", line 878, in __
    return self.__missing__(key)            # support subclasses that define __missing__
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\collections\__init__.py", line 870, in __
    raise KeyError(key)
KeyError: 'debug'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\mohierf\AppData\Roaming\Python\alignak_app\bin\alignak-app.py", line 24, in <module>
    from alignak_app.app import AlignakApp
  File "d:\github\alignak-app\alignak_app\app.py", line 48, in <module>
    create_logger(logger)
  File "d:\github\alignak-app\alignak_app\utils.py", line 84, in create_logger
    if app_config.getboolean('Alignak-App', 'debug'):
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\configparser.py", line 826, in getboolean
    raw=raw, vars=vars, fallback=fallback, **kwargs)
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\configparser.py", line 806, in _get_conv
    **kwargs)
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\configparser.py", line 800, in _get
    return conv(self.get(section, option, **kwargs))
  File "C:\Users\mohierf\AppData\Local\Programs\Python\Python35\lib\configparser.py", line 789, in get
    raise NoOptionError(option, section)
configparser.NoOptionError: No option 'debug' in section: 'Alignak-App'

```

**Comment by mohierf** [comment]
This problem happened because I did not updated my settings.cfg file! You should set default values for all the configuration parameters and catch the loading exceptions :wink:

**Comment by algorys** [comment]
Repoduced and fixed by 5ed8fae:

- Catch error
- Replace by default value
- Sending errors in log file
